packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -47,6 +47,10 @@ const tdoInfoQueryClause = `id
     primaryAsset(assetType: "media") {
       id
       uri
+    }
+    streams {
+      protocol
+      uri
     }`;
 
 function* finishLoadEngineCategories(widgetId, result, { warning, error }) {
```

Comment:
i'll go over the changes needed in the saga file as part of the media-details-page review.